Stripe/PublicHeaders/STPAPIClient.h

```diff
@@ -30,10 +30,12 @@ static NSString *const STPSDKVersion = @"18.3.0";
 @interface Stripe : NSObject FAUXPAS_IGNORED_ON_LINE(UnprefixedClass);
 
 /**
- Set your Stripe API key with this method. New instances of STPAPIClient will be initialized with this value. You should call this method as early as
```

Comment:
why did this behavior change?

Reply:
Hmm. I didn't deliberately keep this behavior, but I don't see why we shouldn't, and it will be less breaking.  Will re-add.